code/modules/surgery/bodyparts/dismemberment.dm

```diff
@@ -352,6 +362,17 @@
 
 	..()
 
+/obj/item/bodypart/proc/synchronize_bodytypes(mob/living/carbon/C)
+	if(!C.dna?.species)
+		return
+	//This codeblock makes sure that the owner's bodytype flags match the flags of all of it's parts.
+	var/all_limb_flags = list()
+	for(var/obj/item/bodypart/BP in C.bodyparts)
```

Comment:
This will include null entries but I'm really hoping `bodyparts` never has null entries.

Reply:
> 
> 
> This will include null entries but I'm really hoping `bodyparts` never has null entries.

Something has gone HORRIBLY wrong if a bodypart is null.